internal/mode/static/manager.go

```diff
@@ -415,23 +436,90 @@ func createTelemetryJob(
 
 	worker := telemetry.CreateTelemetryJobWorker(logger, exporter, dataCollector)
 
-	// 10 min jitter is enough per telemetry destination recommendation
-	// For the default period of 24 hours, jitter will be 10min /(24*60)min  = 0.0069
-	jitterFactor := 10.0 / (24 * 60) // added jitter is bound by jitterFactor * period
-
 	return &runnables.Leader{
 		Runnable: runnables.NewCronJob(
 			runnables.CronJobConfig{
 				Worker:       worker,
 				Logger:       logger,
 				Period:       cfg.TelemetryReportPeriod,
-				JitterFactor: jitterFactor,
+				JitterFactor: telemetryJitterFactor,
 				ReadyCh:      readyCh,
 			},
 		),
 	}
 }
 
+func createUsageReporterJob(
```

Comment:
Manager.go is getting long. What do you think about pulling out all the create job functions into a separate file?

Reply:
I've pulled out some of the code from here, reducing its size, but left the job creation. Still think we should pull the rest of the code out?

Reply:
This looks good to me